src/router/handlers/exportHandler.ts

```diff
@@ -16,11 +16,11 @@ export default class ExportHandler {
         return this.bulkDataAccess.initiateExport(initiateExportRequest);
```

Comment:
So looking at the code for bundles it looks like AuthZ still happens in the routing layer: https://github.com/awslabs/fhir-works-on-aws-routing/blob/mainline/src/router/bundle/bundleHandler.ts#L127

Thoughts on mimicing this approach?